### ISSUE TITLE
Topographical disorientation

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11710]
**Preferred term label:**
Topographical disorientation

**Synonyms**
Impaired wayfinding

**Definition (free text, please give PubMed ID)**
An impaired capacity to navigate through large, real-world spaces, including routes and surroundings the person has used before, that cannot be explained by a failure to recognize objects or landmarks alone.
[PMID: 10468502](https://pubmed.ncbi.nlm.nih.gov/10468502/)

**Parent term (use hpo.jax.org/app)**
Cognitive impairment (HP:0100543) - Would me my best guess. Here it would be a sibling of Confusion and Memory impairment.

However, we have Left-right disorientation (HP:6000000) under Abnormal experience of reality 
HP:5200423, which seems close.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784

**Comment by pnrobinson** [comment]
Not sure if this is the best term, it is more of an inferrence. Let's think about using more strictly definable terms such as landmark agnosia?
